src/app/services/services.module.ts

```diff
@@ -0,0 +1,126 @@
+import { PlatformUtilsService as PlatformUtilsServiceAbstraction } from "jslib-common/abstractions/platformUtils.service";
+import { StateService as StateServiceAbstraction } from "jslib-common/abstractions/state.service";
+import { StateMigrationService as StateMigrationServiceAbstraction } from "jslib-common/abstractions/stateMigration.service";
+import { StorageService as StorageServiceAbstraction } from "jslib-common/abstractions/storage.service";
+import { SystemService as SystemServiceAbstraction } from "jslib-common/abstractions/system.service";
+import { ClientType } from "jslib-common/enums/clientType";
+import { StateFactory } from "jslib-common/factories/stateFactory";
+import { GlobalState } from "jslib-common/models/domain/globalState";
+import { SystemService } from "jslib-common/services/system.service";
+import { ElectronCryptoService } from "jslib-electron/services/electronCrypto.service";
+import { ElectronLogService } from "jslib-electron/services/electronLog.service";
+import { ElectronPlatformUtilsService } from "jslib-electron/services/electronPlatformUtils.service";
+import { ElectronRendererMessagingService } from "jslib-electron/services/electronRendererMessaging.service";
+import { ElectronRendererSecureStorageService } from "jslib-electron/services/electronRendererSecureStorage.service";
+import { ElectronRendererStorageService } from "jslib-electron/services/electronRendererStorage.service";
+
+import { Account } from "../../models/account";
+import { I18nService } from "../../services/i18n.service";
+import { LoginGuardService } from "../../services/loginGuard.service";
+import { NativeMessagingService } from "../../services/nativeMessaging.service";
+import { PasswordRepromptService } from "../../services/passwordReprompt.service";
+import { StateService } from "../../services/state.service";
+import { SearchBarService } from "../layout/search/search-bar.service";
+
+import { InitService } from "./init.service";
+
+@NgModule({
+  imports: [JslibServicesModule],
+  declarations: [],
+  providers: [
+    InitService,
+    NativeMessagingService,
+    SearchBarService,
+    LoginGuardService,
+    {
+      provide: APP_INITIALIZER,
+      useFactory: (initService: InitService) => initService.init(),
+      deps: [InitService],
+      multi: true,
+    },
+    {
+      provide: STATE_FACTORY,
+      useValue: new StateFactory(GlobalState, Account),
+    },
+    {
+      provide: CLIENT_TYPE,
+      useValue: ClientType.Desktop,
+    },
+    { provide: LogServiceAbstraction, useClass: ElectronLogService, deps: [] },
+    {
+      provide: PlatformUtilsServiceAbstraction,
+      useClass: ElectronPlatformUtilsService,
+      deps: [
+        I18nServiceAbstraction,
+        MessagingServiceAbstraction,
+        CLIENT_TYPE,
+        StateServiceAbstraction,
+      ],
+    },
+    {
+      provide: I18nServiceAbstraction,
+      useFactory: (window: Window) => new I18nService(window.navigator.language, "./locales"),
```

Comment:
Could we use a variable for `locales`?

Reply:
I assume you mean token - done, depends on https://github.com/bitwarden/jslib/pull/783

Reply:
Yep. What do you think of also doing the same for `window.navigator.language`? Would let us remove the factory here completely.

Reply:
I didn't know whether it would be going overboard, but I'm happy to :D this is done in the jslib PR and will be added here shortly